contracts/Commitment/Commitment.sol

```diff
@@ -413,6 +527,18 @@ contract Commitment is
     {
         // Validate
         require(investor != 0x0);
+
+        // AUDIT[CHF-34] Undocumented enum feature used.
```

Comment:
That is true. I'd rather have explicit semantics that map enums to values. This is also the reason we have strict requirement on `solc` version for compilation and testing.